app/src/main/java/com/github/se/eventradar/model/repository/event/FirebaseEventRepository.kt

```diff
@@ -8,16 +8,22 @@ import com.google.firebase.firestore.FirebaseFirestore
 import com.google.firebase.firestore.FirebaseFirestoreException
 import com.google.firebase.firestore.ktx.firestore
 import com.google.firebase.ktx.Firebase
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
 import kotlinx.coroutines.channels.awaitClose
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.callbackFlow
 import kotlinx.coroutines.tasks.await
 
 class FirebaseEventRepository(val db: FirebaseFirestore = Firebase.firestore) : IEventRepository {
   private val eventRef: CollectionReference = db.collection("events")
+  private val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm")
+  private val currentDateTimeString: String
+    get() = LocalDateTime.now().format(formatter)
 
   override suspend fun getEvents(): Resource<List<Event>> {
-    val resultDocument = eventRef.get().await()
+    val currentDateTimeString = currentDateTimeString
```

Comment:
Suggestion: Consider deleting this line. 

Note: This applies to all functions below.